os/arch/arm/Kconfig

```
@@ -275,6 +275,7 @@ config ARCH_CORTEXA32
 	select ARCH_USE_MMU
 	select ARCH_HAVE_THREAD_LOCAL
 	select ARCH_ARMV7A_FAMILY
+	select ARCH_HAVE_TESTSET
```

How about adding dependency on smp here?
```suggestion
	select ARCH_HAVE_TESTSET if SMP
```

---

In https://github.com/Samsung/TizenRT/blob/master/os/kernel/Kconfig#L290-L294, this is one of the dependency for having SMP, which means ARCH_HAVE_TESTSET -> CONFIG_SMP, if adding as you suggested, it becomes CONFIG_SMP -> ARCH_HAVE_TESTSET

---

Thanks for pointing it out.

---

Here is some explanation on why we need this macro: https://github.com/Samsung/TizenRT/pull/6040#discussion_r1476235756